src/rgw/CMakeLists.txt

```diff
@@ -401,6 +405,8 @@ add_library(rgw_admin_user SHARED
     ${librgw_admin_user_srcs}
     $<TARGET_OBJECTS:rgw_common>)
 
+target_include_directories(rgw_admin_user PUBLIC "${CMAKE_SOURCE_DIR}/src/rgw")
```

Comment:
@pritha-srivastava just checked `master` and I don't find any trace of `rgw_admin_user` there... Should this be fixed in `master` too and then backported together?

Reply:
Thats the reason why it doesnt fail in master, since librgw_admin_user is not there in master, but is there in octopus. There are no failures in master as far as i know.

Reply:
@pritha-srivastava Would it make sense to cherry-pick c89b59428e66ae6d89d1fd27a4280045c57251cb into octopus instead of adding a special-purpose commit?

@cbodley Is the "unused and not worth the effort to maintain" librgw_admin_user needed in octopus?

Reply:
@cbodley is in a better position to comment on this.

Reply:
> @cbodley Is the "unused and not worth the effort to maintain" librgw_admin_user needed in octopus?

kill it!

> @pritha-srivastava Would it make sense to cherry-pick [c89b594](https://github.com/ceph/ceph/commit/c89b59428e66ae6d89d1fd27a4280045c57251cb) into octopus instead of adding a special-purpose commit?

yes please

Reply:
@smithfarm : Should I cherry-pick https://github.com/ceph/ceph/commit/c89b59428e66ae6d89d1fd27a4280045c57251cb as part of this PR? I will do away with the fix that I have added.

Reply:
Yes, @pritha-srivastava . Please drop your fix and cherry-pick Casey's commit instead.

Reply:
@smithfarm : Done.